app/controllers/subjects_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SubjectsController < ApplicationController
+  def show
+    @pagy, @subject = pagy Subject.search(params[:search]),
```

Comment:
index thì tên biến để số nhiều em nhé
```suggestion
    @pagy, @subjects = pagy Subject.search(params[:search]),
```